packages/auto-icons/src/index.ts

```diff
@@ -42,20 +85,28 @@ export default defineWxtModule<AutoIconsOptions>({
     });
 
     wxt.hooks.hook('build:done', async (wxt, output) => {
-      const image = sharp(resolvedPath).png();
+      let image: sharp.Sharp;
+
+      if (extname(resolvedPath).toLowerCase() === '.png') {
+        // If the file is already PNG, just read it without conversion
+        image = sharp(resolvedPath).png();
+      } else {
+        // If it's not PNG, convert it
+        image = sharp(resolvedPath).toFormat('png').png();
```

Comment:
As for the framework, if it's not png (transparent), it might be better to log a warning. Not a sure opinion, though.

Reply:
That's a good addition!